var/spack/repos/builtin/packages/parmetis/enable_external_metis.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index ca945dd..1bf94e9 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -33,7 +33,7 @@ include_directories(${GKLIB_PATH})
+ include_directories(${METIS_PATH}/include)
+
+ # List of directories that cmake will look for CMakeLists.txt
+-add_subdirectory(${METIS_PATH}/libmetis ${CMAKE_BINARY_DIR}/libmetis)
++#add_subdirectory(${METIS_PATH}/libmetis ${CMAKE_BINARY_DIR}/libmetis)
```

Comment:
This should not be a mac-related issue. We do the same thing here https://github.com/davydden/homebrew-dealiisuite which is tested on Ubuntu 14.04 and OS-X. As you see from the patch here https://github.com/davydden/homebrew-dealiisuite/blob/master/parmetis.rb , we did not have to do any extra things to link. I am quite puzzled why you experience this linking error. Can you try uninstall `metis`, maybe some glitches somewhere?